components/setup-assistant/__examples__/card.jsx

```diff
@@ -180,8 +84,92 @@ class Example extends React.Component {
 							variant="light"
 						/>
 					}
-					steps={this.state.steps}
-				/>
+				>
+					<SetupAssistantStep
+						description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+						estimatedTime="4 mins"
+						heading="Add Users to Your Org"
+						isExpandable
+						progress={100}
+						progressIndicator={
+							<ProgressIndicator
+								completedSteps={subStepsComplete}
+								orientation="vertical"
+								steps={subStepsComplete}
+								variant="setup-assistant"
+							/>
+						}
+					/>
+					<SetupAssistantStep
+						description="Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat."
+						estimatedTime="10 mins"
+						heading="Create Profiles for Your Users"
+						isExpandable
+						progress={33}
+						progressIndicator={
+							<ProgressIndicator
+								completedSteps={subSteps[0]}
+								orientation="vertical"
+								steps={subSteps}
+								selectedStep={subSteps[1]}
+								variant="setup-assistant"
+							/>
+						}
+						scopedNotification={
```

Comment:
I'm still unsure about this prop.

Reply:
Going to remove this and `progressIndicator` and just go with `onRenderContent`